Cargo.lock

```diff
@@ -1,17 +1,17 @@
 [[package]]
```

Comment:
Do you want git to track this lock file?

Reply:
Yes, I think for binaries (as opposed to libraries) that's common: https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries